source/clear-linux/tutorials/dlrs/dlrs.rst

```diff
@@ -196,6 +196,40 @@ We used `flannel`_ as the network provider for these tests. If you
 prefer a different network layer, refer to the Kubernetes
 `networking documentation`_ for setup.
 
+Kubectl
+=======
+
+You can use kubectl to run commands against your Kubernetes cluster.  Refer to the `kubectl overview`_ for details on syntax and operations. Once you have a working cluster on Kubernetes, you can use the following YAML script to start a pod with a simple shell script, and keep the pod open.
```

Comment:
Change "you can use the following YAML" to "use the following yaml"

Reply:
done!